pkg/operation/controlplanesecrets/secrets_deployer.go

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Probably this whole package should be moved to `pkg/operation/botanist/shootsecrets` (some secrets are not related to the control plane, hence, `controlplanesecrets` is not entirely accurate).